core/src/main/java/com/yahoo/oak/DirectUtils.java

```diff
@@ -13,26 +13,32 @@
 import java.nio.Buffer;
 import java.nio.ByteBuffer;
 
-final class UnsafeUtils {
+public final class DirectUtils {
```

Comment:
As this is public, I suggest to add some comments here about the usage and also about Java version required. Maybe some very short mentioning in the README as well.

Reply:
To the best of my knowledge, this should work on any version as long as Oak was compiled on Java8.

Reply:
So just add a comment about it, some users may question about the possible usage

Reply:
I added documentation